tests/suit/utils/plan.go

```diff
@@ -26,8 +26,27 @@ func CreatePlanFromDefinition(cl crclient.Client, def *forkliftv1.Plan) error {
 	}
 	return err
 }
+func NewPlanWithName(namespace string, providerIdentifier forkliftv1.Provider, planName string, storageMap string, networkMap string, vmName []string) *forkliftv1.Plan {
```

Comment:
how about NewPlanWithVmName and NewPlanWithVmId below?

Reply:
good point